### DISCUSSION TITLE
Creating puzzle board

### DISCUSSION THREAD

**Discussion by Kanchana-Sara** [flbulgarelli/headbreaker#85] [Ideas]
Hi I’m creating a puzzle board for the puzzle pieces to fit into. The puzzle board’s should should follow the puzzle created by headbreaker. How can retrieve the svg or is there any easy and better idea on how to do this. Only in the puzzle board I can drop the piece accurately like in grid. Now I only have a canvas and puzzle pieces.

**Comment by flbulgarelli** [comment]
Closing as duplicate of https://github.com/flbulgarelli/headbreaker/discussions/86
